Kumi.Desktop/Program.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using Kumi.Game;
 using osu.Framework;
+using osu.Framework.Development;
+using Squirrel;
 
 namespace Kumi.Desktop;
 
 public static class Program
 {
+#if DEBUG
+    public const string APP_NAME = "Kumi Development";
+#else
+    public const string APP_NAME = "Kumi";
+#endif
```

Comment:
Should probably propagate this over to `KumiGame`.

Reply:
what for

Reply:
...For the title?